yaetos/oracle_sql_job.py

```diff
@@ -19,7 +19,7 @@ def transform(self, **ignored):
         self.OUTPUT_TYPES = self.get_output_types_from_sql(sql)
         cred_profiles = Cred_Ops_Dispatcher().retrieve_secrets(self.jargs.storage)
 
-        print "Running query: \n", sql
+        print("Running query: \n", sql)
```

Comment:
Why do we even print? Should we not be just log?

Reply:
Yes, thanks for the fix @AlejandroUPC . 
And correct, it would be even better using the logging functionality that is already there, replacing the line by  `self.logger.info("Running query: \n {}".format(sql))`. I try to avoid print statements now but some may still be there.

Reply:
If you want to replace it by the logging code above, let me know and I will wait and approve it. If not, I am good to merge your PR as is since it is already an improvement.